shared/fs/banner/reset-banner/container.tsx

```diff
@@ -11,15 +10,17 @@ import {folderNameWithoutUsers} from '../../../util/kbfs'
 import Banner, {getHeight} from '.'
 import * as RowTypes from '../../browser/rows/types'
 
-type OwnProps = {|path: Types.Path|}
+type OwnProps = {
+  path: Types.Path
+}
 
 const mapStateToProps = (state, {path}: OwnProps) => ({
   _tlf: Constants.getTlfFromPath(state.fs.tlfs, path),
   _username: state.config.username,
 })
 
 const mapDispatchToProps = dispatch => ({
-  _onOpenWithoutResetUsers: (currPath: Types.Path, users: {[string]: boolean}) => {
+  _onOpenWithoutResetUsers: (currPath: Types.Path, users: {[K in string]: boolean}) => {
```

Comment:
what's the `K in` doing?

Reply:
Also from flowToTs. Although I just tried; without `K in` it wouldn't work. I think this is [Mapped types](https://www.typescriptlang.org/docs/handbook/advanced-types.html#mapped-types): https://www.typescriptlang.org/docs/handbook/advanced-types.html#mapped-types